resources/.ktlint_editorconfig

```diff
@@ -4,3 +4,11 @@ ktlint_standard_max-line-length = 160
 ktlint_standard = enabled
 ktlint_standard_multiline-expression-wrapping = disabled
 ktlint_standard_string-template-indent = disabled
+ktlint_standard_chain-method-continuation = disabled
+ktlint_standard_class-signature = disabled
+ktlint_standard_trailing-comma-on-call-site = disabled
```

Comment:
:thinking: I actually think the KtLint argument for this rule is somewhat persuasive. Would be nice if IntelliJ actually auto-formatted this...(doesn't, I guess?)

Reply:
The trailing comma rule? Can't see what line you're commenting on in particular

Reply:
If so, I don't think the arguments they provide outweigh the uglyness it introduces to the code 😅

Reply:
I also find trailing commas ugly and not worth the tiny improvement sometimes gained in diffs

Reply:
Beauty is relative. Their argument (less distracting diff changes) isn't.

Reply:
Trailing commas make 'distracting' code in much the same way their absence makes distracting diff changes 😉

Reply:
Whether or not beauty is subjective could be discussed, but either way the improvement to diffs is insignificant and not worth changing coding habits over in my opinion